src/sentry/workflow_engine/migrations/0051_migrate_remaining_issue_alerts.py

```diff
@@ -2185,10 +2182,6 @@ def _create_workflow_actions(if_dcg: Any, actions: list[dict[str, Any]]) -> None
         DataConditionGroupAction.objects.bulk_create(dcg_actions)
 
     # EXECUTION STARTS HERE
-    backfill_key = "backfill_workflow_engine_remaining_issue_alerts"
-    redis_client = redis.redis_clusters.get(settings.SENTRY_MONITORS_REDIS_CLUSTER)
-    progress_id = int(redis_client.get(backfill_key) or 0)
-
```

Comment:
Might be good to keep this now. It'll work fine since we're iterating the whole table, and if you have to cancel at least you can restart from the same place